qrenderdoc/Windows/TextureViewer.cpp

```diff
@@ -4134,6 +4142,38 @@ void TextureViewer::reloadCustomShaders(const QString &filter)
   }
 }
 
+QList<QDir> TextureViewer::getShaderDirectories() const
+{
+  QList<QDir> dirs;
+  dirs.reserve(int(m_Ctx.Config().TextureViewer_ShaderDirs.size() + 1u));
+  dirs.append(QDir(configFilePath(QString())));
+  for(size_t i = 0; i < m_Ctx.Config().TextureViewer_ShaderDirs.size(); i++)
+  {
+    dirs.append(QDir(m_Ctx.Config().TextureViewer_ShaderDirs[int(i)]));
```

Comment:
This cast to `int` is not necessary, `rdcarray` uses `size_t` for indexing to match the STL containers.

Reply:
Got rid of the iteration over index completely.